files/var/www/ks/bootstrap_simp_client

```diff
@@ -111,13 +111,13 @@ EOM
 
   def disable_service(service_name)
     info(title("Disabling #{service_name} service", 2))
-    result = execute("puppet resource service #{service_name} enable=false")
+    result = execute("/opt/puppetlabs/bin/puppet resource service #{service_name} enable=false")
```

Comment:
* Please make `/opt/puppetlabs/bin/puppet` a class constant (e.g., `PUPPET`) or variable (e.g. `@puppet`)
* The `if File.exists?('/opt/puppetlabs/bin/puppet` logic on lines 18-22 no longer make sense if you are assuming a specific path for the puppet executable.  Please fix.